src/app/components/board/index.tsx

```diff
@@ -41,8 +41,8 @@ const _Board = (props: BoardProps) => {
 
 const Board = (props: BoardProps) => {
   const { questions } = props;
-  const { course } = useOfficeHour();
-  const topics = [SELECT_ALL, ...Object.keys(course.tags).sort()];
+  const { course } = useOfficeHourStore();
+  const topics = [SELECT_ALL, ...Object.keys(course?.tags || {}).sort()];
```

Comment:
This is for some unrelated consideration, we should probably default tags for courses (in case some TA deletes all the tags).